model/src/test/kotlin/JsonSchemaTest.kt

```diff
@@ -60,6 +60,19 @@ class JsonSchemaTest : StringSpec() {
                 errors should beEmpty()
             }
         }
+
+        "curation.yml example validates" {
```

Comment:
Nit: All other tests say "validates successfully", so we should probably do so here, too.